sdk/ochopod/models/piped.py

```diff
@@ -381,6 +384,31 @@ def on(self, data):
         self.commands.popleft()
         return 'spin', data, 0
 
+    def log_proc_out(self, proc):
```

Comment:
no need to declare an explicit function .. just define a callback on the fly and reference it (it won't access anything in the actor anyway but rather just log to the global ochopod log)..